site/class-1-load/laurens-booij/laurens-booij/dinners.txt

```diff
@@ -0,0 +1,3 @@
+2017-09-25 Maaltijdsalade met tonijn
```

Comment:
Nice, but there’s a `laurens-booij` in your `laurens-booij`! Could you remove them?